src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    goForward: function(steps = 1) {
+      if (steps > 0) {
+        this.coords.y += steps;
+      }
+      
+      return robot;
+    },
+    goBack: function(steps = 1) {
```

Comment:
```suggestion
    },
    
    goForward: function(steps = 1) {
      if (steps > 0) {
        this.coords.y += steps;
      }
      
      return robot;
    },
    
    goBack: function(steps = 1) {
```
Don't forget to add empty lines between methods to make your code easier to read